### ISSUE TITLE
Possibility of hiding number of followers administratively

### ISSUE THREAD

**Issue by jprojects** [tootsuite/mastodon#8863]
Many people in private social networks live to increase that counter at any price and I think in the fediverse we care more about quality than quantity and do not give importance to it. Many administrators would deactivate that counter if they could.

* * * *

- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by salvadorpla** [comment]
Yes, this admin option could be useful to avoid “strange” behaviours

**Comment by trwnh** [comment]
@salvadorpla The "hide your network" option as implemented in #7532 is only to hide the lists of followers/following, not the number. This is technically still hiding the network, as it says... it just doesn't hide the metadata about the network.

For what it's worth, I support having an option for users to hide follower/following counts as well, not just administratively. No one knows the size of my email address book, or the number of my RSS subscriptions.

**Comment by salvadorpla** [comment]
@trwnh right, understood, Thanks

**Comment by abishekmuthian** [comment]
I second having an option to hide the follow/follower count at the user level.

It would help prevent determining the value of a person based on an integer like Twitter and also to keep the narcissism under check.

**Comment by d4f5409d** [comment]
I also second it, but in my opinion if a user wants to hide it, it shouldn't only hide for themselves, rather everyone else, who visits the account.

**Comment by d4f5409d** [comment]
Related to #1660
